src/sentry/issues/auto_source_code_config/task.py

```diff
@@ -187,39 +187,22 @@ def create_repos_and_code_mappings(
                 .order_by("-date_added")
                 .first()
             )
-
             if not repository:
+                created = False
```

Comment:
I'm refactoring some of the logic here to make sure the metrics are called correctly.